source/releasenotes/about.rst

```diff
@@ -17,71 +17,30 @@
+• Backup and Object Storage Limits
+• Virtual Trusted Platform Module (vTPM) Support
+• Support for KVM on s390x (IBM Z and LinuxONE) architecture
+• Management Server Maintenance Mode
+• Incremental Volume Snapshots for KVM
+• File-Based Disk-Only Instance Snapshots with KVM
+• Reconciliation of VM and Volume Copy/Migrate operations for KVM agent
+• Storage Access Groups
+• Volume Allocation Algorithm Support
+• VMware to KVM migration improvements
+• Storage plugins - Ceph, PowerFlex, StorPool, Linstor related fixes and improvements
+• Several UI fixes and improvements, including Deploy Instance Form, Announcement Banner, etc
 
 
 The full list of new features can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.20.1.0/releasenotes/changes.html
-
-What's New in 4.20.0.0
-=======================
-
-Apache CloudStack 4.20.0.0 is the initial 4.20 LTS release with 190+ new
-features, improvements and bug fixes since 4.19, including 15 major
-new features. Some of the highlights include:
-
-• Webhooks
-• Dynamic and Static Routing
-• Ceph RGW Object Store Support
-• NSX integration
-• Shared Filesystems
-• Multi-arch Zones
-• Simple NAS backup plugin for KVM
-• Usage UI
-• API documentation in UI
-
-
-The full list of new features can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.20.0.0/releasenotes/changes.html
-
-Log4j Upgrade
-=============
-
-Up until 4.19.x.x, the logging library used for the project was Log4j 1.29. 
-The 4.20.0.0 version has updated the library to Log4j2. The new Log4j2 configuration file format is not backwards 
-compatible with the old one. The 4.20.0.0 packages will come with the default configuration files updated. 
-Users that have made customizations to their files must update their configuration files to match with the new format, 
-the `official Log4j documentation`_ might help you migrate your custom configurations.
-
-JRE Upgrade
-============
-
-Up until 4.19.x.x, the JRE used for ACS was JRE 11. In 4.20.0.0, JRE has been upgraded to JRE 17 as JRE 11 has reached EOL. 
-This means that Centos7 (EL7) is no longer supported.
-
-.. _official Log4j documentation: https://logging.apache.org/log4j/2.x/migrate-from-log4j1.html
-
-Events Message Bus Change
-=========================
-On upgrading from 4.19.x or lower, existing AMQP or Kafka integration
-configurations should be moved from folder
-``/etc/cloudstack/management/META-INF/cloudstack/core`` to
-``/etc/cloudstack/management/META-INF/cloudstack/event``
-
-Guest OS Categories Change
-==========================
-
-The guest operating system categories have been updated in 4.21, resulting in a
-reorganization of the guest operating systems with respect to categories.
-
-If the ``oscategoryid`` functionality for hosts is being used, ensure it is
-pointing to the correct guest operating system category ID.
+https://docs.cloudstack.apache.org/en/4.21.0.0/releasenotes/changes.html
```

Comment:
Do we need to mention issue around, https://github.com/apache/cloudstack/pull/11522?

Reply:
@DaanHoogland can you advise, what's the best place to put this as a known issue

Reply:
updated